src/components/Asset/Selector/Modal/index.tsx

```diff
@@ -35,13 +35,13 @@ const AssetList = ({ assets, onSelect, selected }: AssetListProps): JSX.Element
             type="button"
             size="lg"
             variant="secondary"
-            key={token.address}
+            key={token.id}
             onClick={() => onSelect(token)}
             className="w-full items-center justify-start gap-4 px-3 py-1 border-0 bg-[rgba(0,0,0,.2)] text-left hover:opacity-80 dark:bg-[rgba(255,255,255,.12)]"
           >
             <span className="relative">
-              <Avatar size="xs" letters={token.symbol} src={token.logoURI} shape="circle" className="text-xs" />
-              {selected == token.address && (
+              <Avatar size="xs" letters={token.symbol} /* src={token.logoURI} */ shape="circle" className="text-xs" />
```

Comment:
What are we going to do about the token logo?

Reply:
So on nabla we have a constant with images from coingecko. Maybe there is a better way

Reply:
So you have a hard-coded mapping of token symbol to logo?

Reply:
No logo currently. So in that avatar if the image is missing it defaults to showing the symbol. This is useful even once we have images is one does not load or is missing.